arduino/i2c_slave_code_rccar/i2c_slave_rccar/i2c_slave_rccar.ino

```diff
@@ -89,14 +99,48 @@ bool checkRemote() {
 
 // called when data is sent through i2c to arduino
 // to update the motor values
-void receiveEvent() {
+void receiveEvent(int bytes) {
     // check incoming bytes from master
     uint8_t read_array[2];
+    int i = 0;
     while(Wire.available()) {
         read_array[i] = Wire.read();
+        i++;
```

Comment:
this clearly fixes *one* issue but it could still start trampling the stack if more than two bytes are available. this is safer and nicer:
```c++
for(int i = 0; Wire.available() && i < 2; i++) {
    read_array[i] = Wire.read();
}
```

Reply:
Good spot!